actions/telemetry_actions.jsx

```diff
@@ -14,6 +17,15 @@ const SUPPORTS_MEASURE_METHODS = isSupported([
     performance.clearMeasures,
 ]);
 
+export function isTelemetryEnabled(store = reduxStore) {
```

Comment:
Not a fan of passing the store in but couldn't find a cleaner method that would allow for unit testing.

Reply:
Good call, that's much cleaner. I'll update the PR.